ch12/signal_helper.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __SIGNAL_HELPER_H___
+#define __SIGNAL_HELPER_H___
+
+#include <signal.h>
+/**
+ * setup_altsigstack - Helper function to set alternate stack for sig-handler
+ * @stack_sz:	required stack size
+ *
+ * Return: 0 on success, -ve errno on failure
+ */
+int setup_altsigstack(size_t stack_sz)
+{
+	stack_t ss;
+
+	printf("Alt signal stack size = %zu\n", stack_sz);
+	ss.ss_sp = malloc(stack_sz);
+	if (!ss.ss_sp){
+		FATAL("malloc(%zu) for alt sig stack failed\n", stack_sz);
+		return -ENOMEM;
```

Comment:
remove this return statement; the FATAL() macro causes process exit with exit(EXIT_FAILURE);

Reply:
> Hi Jagadeesh, thanks for your pull request..
> I've made small comments in your function, pl address them first.
> Also, you'd have to complete the patch by actually invoking this func in ch12/handle_segv.c.
> Of course, first pl ensure you test it. Thanks.

Sorry, My fault the invocation part was left over in my local repo.